src/mdformat/_cli.py

```diff
@@ -197,6 +201,13 @@ def make_arg_parser(
     parser.add_argument(
         "--check", action="store_true", help="do not apply changes to files"
     )
+    parser.add_argument(
+        "--no-validate",
+        action="store_false",
+        dest="validate",
+        default=None,
```

Comment:
```suggestion
        action="store_const",
        const=True,
        dest="validate",
```

Reply:
End result is probably the same, but I think this may be more in line with what the other flags do.

Reply:
I think this needs to be `const=False`, but I like that this is more explicit